rplugin/python3/molten/outputbuffer.py

```diff
@@ -218,11 +218,11 @@ def show_virtual_output(self, anchor: Position) -> None:
         )
         self.canvas.present()
 
-    def show_floating_win(self, anchor: Position) -> None:
+    def show_floating_win(self, anchor: Position, offset: int = 0) -> None:
         win = self.nvim.current.window
         win_col = win.col
-        win_row = self._buffer_to_window_lineno(anchor.lineno + 1)
-        if win_row == 0:  # anchor position is off screen
+        win_row = self._buffer_to_window_lineno(anchor.lineno + 1) + offset
+        if win_row <= 0:  # anchor position is off screen
```

Comment:
did this cause problems? i don't think it should have, but if it did that's probably a bug or broken assumption that might break other things, I'd want to look into that

Reply:
It didn't cause any immediate problems, but since the `+ offset` was added and offset is negative, `win_row` could then be negative and this was to catch that possibility.